main.py

```diff
@@ -0,0 +1,65 @@
+from fastapi import FastAPI, Depends, Query
+from fastapi import HTTPException
+from sqlalchemy.orm import Session
+
+import crud
+import schemas
+from crud import get_autor_from_db, get_all_books, create_book_db
```

Comment:
The import statement `from crud import get_autor_from_db, get_all_books, create_book_db` contains a typo in the function name `get_autor_from_db`. It should be `get_author_from_db` to match the correct spelling of 'author' and the function definition in `crud.py`.